libs/repositories/src/UsdRepository/UsdRepositoryCow.ts

```diff
@@ -26,9 +26,11 @@ export class UsdRepositoryCow extends UsdRepositoryNoop {
       return null;
     }
     const erc20 = await this.erc20Repository.get(chainId, tokenAddress);
-    const tokenDecimals = erc20?.decimals || 18;
-    if (tokenDecimals === null) {
-      throw new Error('Token decimals not found for ' + tokenAddress);
+    const tokenDecimals = erc20?.decimals;
+
+    if (tokenDecimals === undefined) {
+      console.info('Token decimals not found for ' + tokenAddress);
```

Comment:
Is there a better logger maybe?

Reply:
We do have better logger. https://github.com/cowprotocol/bff/blob/hackathon-mooo/apps/api/src/logger.ts#L26 

Is this something we really want to log?

Reply:
We are blind when it comes to why a token has no price otherwise.
In fact, right now I can't tell why on prod it returns `315438595.8198187` while on local it returns `0.031570713153867405`.
Yes, the assumed decimals to be 18 play a role, but why it loads no decimals in the first place is the unknown part.

And thanks, I'll use that logger :)

Reply:
I think the key is here 
<img width="770" alt="image" src="https://github.com/user-attachments/assets/859cdf06-7db3-43b2-8ccc-a48bb4018557" />

Thanks for adding the logger

Reply:
Yes. The question that I have is, why did it fail only for decimals?